### ISSUE TITLE
IndexOutOfBoundsException when extracting dependency of jar with Java 9 module-info.class

### ISSUE THREAD

**Issue by OlivierJaquemet** [jeantessier/dependency-finder#4]
The following exceptions occurs when trying to extract `jaxb-api-2.3.0.jar` .

To reproduce : 
- Download [jaxb-api-2.3.0.jar](http://central.maven.org/maven2/javax/xml/bind/jaxb-api/2.3.0/jaxb-api-2.3.0.jar) from https://mvnrepository.com/artifact/javax.xml.bind/jaxb-api/2.3.0
- Start dependency extraction on this jar

Tested with master as of today.

```
Exception in thread "Thread-5" java.lang.IndexOutOfBoundsException: Index: 7444, Size: 6
	at java.util.ArrayList.rangeCheck(ArrayList.java:657)
	at java.util.ArrayList.get(ArrayList.java:433)
	at com.jeantessier.classreader.impl.Classfile.getRawClass(Classfile.java:200)
	at com.jeantessier.classreader.impl.Classfile.getClassName(Classfile.java:204)
	at com.jeantessier.classreader.impl.Classfile.<init>(Classfile.java:104)
	at com.jeantessier.classreader.impl.Classfile.<init>(Classfile.java:74)
	at com.jeantessier.classreader.impl.DefaultClassfileFactory.create(DefaultClassfileFactory.java:41)
	at com.jeantessier.classreader.impl.DefaultClassfileFactory.create(DefaultClassfileFactory.java:1)
	at com.jeantessier.classreader.TransientClassfileLoader.load(TransientClassfileLoader.java:70)
	at com.jeantessier.classreader.ClassfileLoaderEventSource.load(ClassfileLoaderEventSource.java:133)
	at com.jeantessier.classreader.ZipClassfileLoader.load(ZipClassfileLoader.java:127)
	at com.jeantessier.classreader.JarClassfileLoader.load(JarClassfileLoader.java:53)
	at com.jeantessier.classreader.ClassfileLoaderEventSource.load(ClassfileLoaderEventSource.java:93)
	at com.jeantessier.classreader.ClassfileLoader.load(ClassfileLoader.java:55)
	at com.jeantessier.dependencyfinder.gui.DependencyExtractAction.run(DependencyExtractAction.java:89)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by jeantessier** [comment]
This is related to issue #2, except that it's about Java 9 bytecode, this time.

**Comment by jeantessier** [comment]
I put a change on `master` to swallow the exception, like the code was already doing for `IOException`.

**Comment by OlivierJaquemet** [comment]
Bug fixed through https://github.com/jeantessier/dependency-finder/commit/a5de21f465e809015be3d3d73d863dfe8a44d5cd

Thanks again @jeantessier , I just built dependency-finder and could confirm symptoms no longer occurs. Closing this issue.

**Comment by jeantessier** [comment]
Included in release 1.2.1-beta5.
